commands/faas_test.go

```diff
@@ -0,0 +1,53 @@
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestLoadsDefaultYAMLWhenPresent(t *testing.T) {
+	rs := resetState()
+	defer rs()
+	os.Chdir("./test_fixtures")
+
+	Execute([]string{"help"})
+
+	if yamlFile != "stack.yml" {
+		t.Fatal(fmt.Printf("Expected yamlFile to equal %v got %v\n", "stack.yml", yamlFile))
+	}
+}
+
+func TestLoadsFromPametersYAMLWhenGivenAndDefaultPresent(t *testing.T) {
+	rs := resetState()
+	defer rs()
+	os.Chdir("./test_fixtures")
+
+	Execute([]string{"help", "--yaml=myfile.yml"})
+
+	if yamlFile != "myfile.yml" {
+		t.Fatal(fmt.Printf("Expected yamlFile to equal %v got %v\n", "stack.yml", yamlFile))
```

Comment:
Why not `t.Fatalf("...")` instead?

Reply:
Good catch, TBH, I never use the the default testing framework in go, I find you have to write way too much boilerplate.  I generally use this https://github.com/stretchr/testify

Reply:
Thanks for matching the project style 💯